src/main/antlr4/FIRRTL.g4

```diff
@@ -364,5 +364,5 @@ SKIP_
 	;
 
 NEWLINE
-	:'\r'? '\n' ' '*
+	:'\r'? '\n' [' '\t]*
```

Comment:
You're right, that should just be a space. Hahaha this PR will allows you to begin a line with `'` in addition to tabs and spaces.

Your comment made me look into it a little further. If remove the single quotes and you start a line with single quotes, it will do the same issue as describe above and just ignore all lines after the line starting with `''''`. So even with that fix, this PR kind of fixes the problem for tabs but the fundamental issue of weird stuff happening is not fixed. I think this needs to be solved a different way.